src/main/resources/lib/guillotine/index.js

```diff
@@ -65,10 +67,93 @@ function createContext(options) {
     }
     context.options.applications = context.options.applications || [app.name];
     context.options.allowPaths = context.options.allowPaths || [];
+    context.options.subscriptionEventTypes = context.options.subscriptionEventTypes || ['node.*'];
 
     return context;
 }
 
+//──────────────────────────────────────────────────────────────────────────────
+// GraphQL WS Protocol
+//──────────────────────────────────────────────────────────────────────────────
+const graphQlSubscribers = {};
+
+function cancelSubscription(sessionId) {
+    Java.synchronized(() => {
```

Comment:
Create a separate issue, probably in XP. It should be possible to use websoket context for such needs without involving java